### ISSUE TITLE
investigate why LADOT didn't get a report

### ISSUE THREAD

**Issue by hunterowens** [cal-itp/reports#26]
LADOT (DASH and Comuter Express) didn't get a report. 

Here's the entry from `agencies.yml`

```
dash:
  agency_name: DASH
  feeds:
    - gtfs_schedule_url: http://lacitydot.com/gtfs/administrator/gtfszip/ladotgtfs.zip
      gtfs_rt_vehicle_positions_url: null
      gtfs_rt_service_alerts_url: null
      gtfs_rt_trip_updates_url: null
  itp_id: 183

**Comment by machow** [comment]
Ah, they don't include a `feed_info.txt`  file in their feed. Only a few agencies lack this, and right now, that will break report generation. 

In order to fix, we need to...

* Change [this JOIN in the pipeline](https://github.com/cal-itp/data-infra/blob/main/airflow/dags/gtfs_views/reports_gtfs_schedule_index.sql#L79) to be a LEFT JOIN - so we keep agencies w/o feed info files
* Modify the feed_info query in reports.ipynb to produce a 1 row DataFrame with NULL values, and the same columns when the file is missing.

Basically we need to wrap this code in an if-else statement:

```python
# Data ====
# 1. High level feed info ----
feed_info = (
    tbl.gtfs_schedule_type2.feed_info()
    >> filter_end
    >> filter_itp
    >> select_rm_calitp
    >> collect_to_dict
)
```

**Comment by thekaveman** [comment]
Just a note/FYI, as mentioned over in cal-itp/mobility-marketplace#296:

LADOT gave us some feedback that they would rather be known as "Los Angeles Department of Transportation" instead of "DASH" on that site. It looks like the value has been updated in the [Primary List of CA Transit Providers](https://docs.google.com/spreadsheets/d/105oar4q_Z3yihDeUlP-VnYpJ0N9Mfs7-q4TnribqYLU). Unsure if this is related, but coincidental at least?

**Comment by hunterowens** [comment]
I think we should make it as an update to agencies.yml as well. I'll go in
and change it.

**Comment by machow** [comment]
Looking at this with @Nkdiaz, we have some edge cases to decide. For example...

* South County Area Transit (cal itp 388) was in our agencies.yml, but we never successfully downloaded files for it, and later removed it.
* currently, our reports index will not generate anything for it because it does not have feed_info.
* if we enable report generation for feeds with no feed_info, we will start generating reports for it, even though we never downloaded or parsed any of its data (since our data indicates we had it in agencies.yml). The report generation should work, but will be mundane (e.g. 0 service hours each day, no feed info, etc..)

```python
from calitp.tables import tbl
from siuba import *

tbl.views.reports_gtfs_schedule_index() >> filter(_.calitp_itp_id == 388)

# We just failed to extract it for 103 days ----

(tbl.views.gtfs_schedule_fact_daily_feeds()
  >> inner_join(_, tbl.views.gtfs_schedule_dim_feeds(), "feed_key")
  >> filter(_.calitp_itp_id == 388)
  >> select(_.calitp_feed_name, _.date, _.extraction_status)
  >> count(_.extraction_status)
)
```

**Comment by evansiroky** [comment]
I believe we should be generating reports for all operators - even those that don't have a feed to speak of in the first place. We should definitely include some custom messaging in cases where a feed does not exist or there is some other major problem.

**Comment by acouch** [comment]
This has been fixed with #161
